### ISSUE TITLE
Add support for web.Release.config

### ISSUE THREAD

**Issue by mikeblakeuk** [tompazourek/AspNetMvcOnSdk#1]
can we add support for 

    <TransformXml Source="Web.config" Transform="Web.$(Configuration).config" Destination="$(IntermediateOutputPath)$(TargetFileName).config" />

?

**Comment by tompazourek** [comment]
@mikeblakeuk Yes, this could be interesting. I never really used the built-in XML transformations functionality and always had something custom-built (some script or tool or something).

This transformation happens during web publishing, and if you do ordinary builds in VS, there's no transformation at all. Is that right?

**Comment by mikeblakeuk** [comment]
with
`  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk.Publish" />`
